packages/flutter_tools/test/general.shard/resident_runner_test.dart

```diff
@@ -445,6 +446,25 @@ void main() {
     expect(cacheDill, isNot(exists));
   }));
 
+  test('ResidentRunner can run source generation', () => testbed.run(() async {
```

Comment:
Should we also write a test testing when an error has occurred?

Reply:
Done